pycape/cape.py

```diff
@@ -273,6 +276,7 @@ def run(
         self,
         function_ref: Union[str, fref.FunctionRef],
         *args: Any,
+        pcrs=None,
```

Comment:
Can you add typing for `pcrs` as you did  for `_key` and `bootstrap` ? thank you!